server/games/game.py

```diff
@@ -136,11 +136,12 @@ def __init__(self, id, game_service, game_stats_service,
 
         self.mods = {}
         self._logger.info("{} created".format(self))
-        asyncio.get_event_loop().call_later(20, self.timeout_game)
+        asyncio.get_event_loop().create_task(self.timeout_game())
 
-    def timeout_game(self):
+    async def timeout_game(self):
+        asyncio.sleep(20)
```

Comment:
Simply calling `asyncio.sleep(20)` doesn't actually cause a sleep, you need to `await` it :)